app/src/main/java/a3/com/convo/activities/LoginActivity.java

```diff
@@ -165,7 +177,7 @@ public void onCompleted(
                                         public void done(ParseException e) {
                                             if (e == null) {
                                                 Log.e("LoginActivity", "Create page success");
-                                                user.add("pageLikes", newPage.getObjectId());
+                                                user.add(Constants.PARSE_PAGE_LIKES_KEY, newPage.getObjectId());
```

Comment:
What if newPage is null? Same as above. Sync up with @alveerak and change

Reply:
I don't think newPage would be null because at the very least the Page.newInstance() method returns an empty Page

Reply:
Then this looks good.